### PR TITLE
Prepare for updated content API models

### PR DESCRIPTION
**Pull request by tonytw1**

### CODE REVIEW

File: client-default/version.sbt [hunk 1]
```diff
@@ -1 +1,2 @@
-version in ThisBuild := "17.2-SNAPSHOT"
+// TODO: is it?
+version in ThisBuild := "IRRELEVANT"
```
Comment (tonytw1) [inline client-default/version.sbt:2]: Do we want to include this? Probably can.

**Comment by JustinPinner** [comment]
Hm. I hadn't planned to open this PR just yet which is why those version changes are still there. I'd like to understand if we _can_ remove those versions or if they are in fact used by anything when running the full release cycle.

**Comment by JustinPinner** [comment]
Hey @tonytw1 I think this is where we needed to be with this. Does it look how you expected? I left the TODOs in there for the time being.

**Comment by JustinPinner** [comment]
For reference, this change relates to this models update: https://github.com/guardian/content-api-models/pull/188

**Review by JustinPinner — APPROVED** [review]
Again, I'm approving changes I had a hand in creating, but that's how we're rolling these days it seems. 👍  (Should be harmless, right? 🤔 )

**Review by juanvaccari — APPROVED** [review]

File: project/Dependencies.scala [hunk 2]
```diff
@@ -3,7 +3,7 @@ import sbt._
 object Dependencies {
   val scalaVersions = Seq("2.11.12", "2.12.10", "2.13.1")
 
-  val CapiModelsVersion = "15.9.8"
+  val CapiModelsVersion = "15.9.10"
```
Comment (JustinPinner) [inline project/Dependencies.scala:6]: Based on https://github.com/guardian/content-api/pull/2556/files#diff-835bedc1586a84c60f1a1bf7f86a508d2494a56e65459b5abcbec31b15fc035d I guess we ought to bump this up to `15.9.12` too?

Comment (JustinPinner) [reply project/Dependencies.scala:6]: I'll do that and see how we fare...

**Pull request merged**